repairnator/scripts/launch_dockerpool.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -e
+
+# Use to create args in the command line for optional arguments
+function ca {
+  if [ -z "$2" ];
+  then
+      echo ""
+  else
+    if [ "$2" == "null" ];
+    then
+        echo ""
+    else
+        echo "$1 $2 "
+    fi
+  fi
+}
+
+if [ "$#" -eq 0 ]; then
```

Comment:
If there should be exactly one argument, maybe use `-ne 1` instead of `-eq 0`

Reply:
Actually `launch_dockerpool.sh` needs to receive one arg (the list of builds), but it can also receive one more arg if it's running throught `launch_repairnator.sh`.

Reply:
> it can also receive one more arg

Which one? I cannot see it in `launch_repairnator.sh`

Reply:
Now `launch_dockerpool.sh` receives only one arg, the list of builds. The other one was to control if the things had been initialized, but with your approach we don't need it anymore.

Reply:
So we can use `-ne 1` here...

Reply:
Done.